statsmodels/genmod/generalized_linear_model.py

```diff
@@ -877,6 +878,23 @@ def fit(self, start_params=None, maxiter=100, method='IRLS', tol=1e-8,
             values for gradient optimization.  Only relevant if
             `method` is set to something other than 'IRLS'.
 
+        IRLS Parameters
```

Comment:
Whoops. Didn't realize that was a "thing". I presume you could use any header. Thanks for that link. I tried to follow